README.md

```diff
@@ -0,0 +1,40 @@
+![gusto logo](./assets/Gusto_logo.png)
+
+# Gusto Python Client
+**Payroll, HR, benefits. Simplified**
+
+> [!NOTE]
+> Exciting news! Our first Python library beta release is here.
+
+## Authentication
+
+Gusto Embedded supports two types of tokens that are used throughout development: company-level tokens and system-level tokens.
+
+System access tokens allow you to carry out application level actions such as:
+* Creating a partner managed company
+* Viewing invoices
+
+Company access tokens allow you to carry out company level actions, such as:
+* Getting company employees
+* Running payroll for a company
+
+Each partner managed company has a single access token that cannot be used to access other companies.
+
+See the [System Access Tokens](https://docs.gusto.com/embedded-payroll/docs/system-access-tokens) and [Company Access Tokens](https://docs.gusto.com/embedded-payroll/docs/company-access-tokens) for more details.
+
+## Client Libraries
+
+<!-- Start Gusto Python Client Libraries -->
+| Library | Description | PyPI |
+| :- |:- |:- |
+| **[Gusto Embedded](https://github.com/Gusto/gusto-python-client/tree/main/gusto_embedded)** | Client library for the Gusto Embedded API | [![PyPI](https://img.shields.io/pypi/v/gusto-embedded.svg?color=%230A8080)](https://pypi.org/project/gusto_embedded) |
```

Comment:
Add to description – why to use each library 
- use this if you're building embedded payroll
- use this if you're building a Gusto integration